src/screens/AdminPortal/OrganizationFundCampaign/modal/CampaignModal.module.css

```
@@ -1,55 +1,436 @@
+    var(--campaign-modal-header-padding-right)
+    var(--campaign-modal-header-padding-bottom)
+    var(--campaign-modal-header-padding-left);
+  border-bottom: none;
 }
 
-.noOutline input:disabled {
-  -webkit-text-fill-color: var(--color-gray-600);
+.campaignModal :global(.modal-title) {
+  margin: 0;
+  color: var(--color-black);
+  font-family: 'Lato', sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Lato" (font-family-name-quotes)

(font-family-name-quotes)